content/resources/registry/publishing/_index.md

```diff
@@ -0,0 +1,183 @@
+
+### Version Control
+
+Version control is a system for recording changes to files over time, allowing you to track any modifications and
+revert back to previous versions. Version control is great for making collaboration with others seamless and
+manageable.
+
+### Zenodo - DOI generation
+
+Zenodo is an open-source project for generating DOIs that integrates well with GitHub. We recommend using Zenodo for
+generating a permanent DOI (Digital Object Identifier) for your recognizer. A DOI provides a unique and persistent
+identifier for your work, enabling others to cite it in their publications. Obtaining a DOI through Zenodo enhances
+the discoverability and visibility of yopur recognizer contributing to the impact and recoginition of your work
+within the scientific community.
+
+## How to publish your recognizer
+
+We recommend using our [Recognizer Template in GitHub](https://github.com/ecoacoustics/recognizer-template) for
+publishing your recognizer. If you are familiar with GitHub you can proceed directly to the template. However, if you
+need more information on how to use it effectively, please continue reading this article.
+
+### What goes in the repository?
+
+#### One or more recognizers, which should consist of:
+- Code to run the recognizers
+- The code that makes the recognizers
+- Test and training data (or instructions on how to access said data)
+- Models or artifacts (or instructions on how to access said data)
+
+#### README-file that describes:
+
+- What species are detected by the recognizer.
+- The performance of the recognizer.
+- License and attribution information.
+
+### How do I use the GitHub template to publish my recognizer?
+
+#### Step 1.  Create a GitHub account or Log in to GitHub
+
+If you don't have a GitHub account yet,  the first step is to create one. You can create an account in
+[GitHub sign-in page](https://github.com/signup) following the prompts to set up your profile. If you already have a
+GitHub account, log in. 
+
+Learn more and get started with Git and Github:
+- [Signing up for a new GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
+- [Getting started with your GitHub account](https://docs.github.com/en/get-started/onboarding/getting-started-with-your-github-account)
+- [Hello World - quick tutorial on many GitHub features](https://docs.github.com/en/get-started/quickstart/hello-world)
+- [Set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
+- [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/getting-started-with-github-desktop)
```

Comment:
This seems tangential: it really should be in a separate page we link to. Probably the   Help Centre  /  Software / GitHub page

Reply:
We should probably just make this whole section link to that page